src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_txt_reader.py

```diff
@@ -1,50 +1,22 @@
 from __future__ import annotations
 
 from io import StringIO
-import re
 from typing import Any
 
 import pandas as pd
 
 from allotropy.named_file_contents import NamedFileContents
 from allotropy.parsers.beckman_vi_cell_xr.constants import (
     DATE_HEADER,
-    DEFAULT_VERSION,
-    MODEL_RE,
-    XrVersion,
+    HEADINGS_TO_PARSER_HEADINGS,
 )
 from allotropy.parsers.lines_reader import read_to_lines
 
-HEADINGS_TO_PARSER_HEADINGS = {
-    "Dilution": "Dilution factor",
-    "Total cells / ml (x 10^6)": "Total cells/ml (x10^6)",
-    "Total cells": "Total cells",
-    "Average diameter (microns)": "Avg. diam. (microns)",
-    "Viable cells": "Viable cells",
-    "Average circularity": "Avg. circ.",
-    "Viability (%)": "Viability (%)",
-    "Total viable cells / ml (x 10^6)": "Viable cells/ml (x10^6)",
-    "Sample ID": "Sample ID",
-    "Cell type": "Cell type",
-}
-
-NUMERIC_COLUMNS = [
-    "Dilution factor",
-    "Total cells/ml (x10^6)",
-    "Total cells",
-    "Avg. diam. (microns)",
-    "Viable cells",
-    "Avg. circ.",
-    "Viability (%)",
-    "Viable cells/ml (x10^6)",
-]
-
 
 class ViCellXRTXTReader:
     def __init__(self, contents: NamedFileContents) -> None:
         self.lines = read_to_lines(contents)
         self.file_info = self._get_file_info()
-        self.file_version = self._get_file_version()
```

Comment:
double checking, we don't need this anymore because we can handle both `Sample date/time` and `Sample date` now and it wasn't used otherwise?

Reply:
Yup exactly 👍